lib/tapioca/gemfile.rb

```diff
@@ -176,14 +165,52 @@ def exported_rbi_tree
 
       private
 
+      sig { returns(T::Array[Pathname]) }
+      def collect_files
+        if default_gem?
+          # `Bundler::RemoteSpecification` delegates missing methods to
+          # `Gem::Specification`, so `files` actually always exists on spec.
+          T.unsafe(@spec).files.map do |file|
+            resolve_to_ruby_lib_dir(file)
+          end
+        else
+          @spec.full_require_paths.flat_map do |path|
+            Pathname.glob((Pathname.new(path) / "**/*.rb").to_s)
+          end
+        end
+      end
+
       sig { returns(T::Boolean) }
       def default_gem?
         @spec.respond_to?(:default_gem?) && @spec.default_gem?
       end
 
       sig { returns(Pathname) }
       def ruby_lib_dir
-        Pathname.new(RbConfig::CONFIG["rubylibdir"])
+        RbConfig::CONFIG["rubylibdir"]
```

Comment:
Is this method used anywhere other than in `resolve_to_ruby_lib_dir`? Could it be a local variable there?